scripts/get_assets_properties.js

```diff
@@ -0,0 +1,40 @@
+const fetch = require('node-fetch');
+
+const defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI0YjNhNmQ4My01OTdlLTRjNmQtYTllYS1lMjM0NmYxZTU5ZmUiLCJpZCI6MTg3NTIsInNjb3BlcyI6WyJhc2wiLCJhc3IiLCJhc3ciLCJnYyJdLCJpYXQiOjE1NzQ0MTAwNzV9.Cj3sxjA_x--bN6VATcN4KE9jBJNMftlzPuA8hawuZkY';
+
+function fetchIon(url, token=defaultAccessToken) {
+  return fetch(url, {
+    headers: {
+      'Authorization': `Bearer ${token}`
+    }
+  });
+}
+
+const header = `
+/* eslint-disable */
+// DO NOT EDIT: this is file is automatically generated by get_assets_properties script
+`;
+
+// get all the user's assets
+const endpoints = fetchIon('https://api.cesium.com/v1/assets')
```

Comment:
Cesium ION contains assets not used in production.
A better way is to import the `defaultLayerTree` from the `constants.js` file:
-  it is always up-to-date;
- only reference used Cesium ION assets;
- also reference non-Cesium ION assets.

Reply:
I'm trying with this `.mjs` file:
```js
import {defaultLayerTree} from '../src/constants.js';
```
but the error is:
```
(node:786836) ExperimentalWarning: The ESM module loader is experimental.
file:///home/fredj/code/ngm/scripts/boo.mjs:1
import {defaultLayerTree} from '../src/constants.js';
        ^^^^^^^^^^^^^^^^
SyntaxError: The requested module '../src/constants.js' is expected to be of type CommonJS, which does not support named exports. CommonJS modules can be imported by importing the default export.
For example:
import pkg from '../src/constants.js';
const {defaultLayerTree} = pkg;
```
So I've tried with:
```js
import constants from '../src/constants.js';
const {defaultLayerTree} = constants;
```
error is:
```
(node:787164) ExperimentalWarning: The ESM module loader is experimental.
/home/fredj/code/ngm/src/constants.js:1
import Rectangle from 'cesium/Source/Core/Rectangle';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```